packages/documentation/src/stories/components/segmented-button/segmented-button.docs.mdx

```diff
@@ -27,4 +27,9 @@ It allows users to toggle between two or more content sections within the same a
 <Canvas of={SegmentedButtonStories.IconExample} sourceState="shown" />
 <Controls of={SegmentedButtonStories.IconExample} />
 
+## Segmented text and icon button
+
+<Canvas of={SegmentedButtonStories.TextAndIconExample} sourceState="shown" />
+<Controls of={SegmentedButtonStories.TextAndIconExample} />
+
```

Comment:
This example is not placed correctly in the page. Can you check other pages to see where it should be placed? 

Also, we usually add an explanatory text for this kind of examples.